modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/resource/v1_0/MessageBoardThreadResourceImpl.java

```diff
@@ -671,6 +671,12 @@ private void _updateQuestion(
 
 			mbThread.setQuestion(showAsQuestion);
 		}
+
+		Boolean subscribedToMessage = messageBoardThread.getSubscribed();
+
+		if ((subscribedToMessage != null) && subscribedToMessage) {
```

Comment:
Use GetterUtil.getBoolean to avoid this check

Reply:
Done! Thanks! :)